website/next.config.js

```diff
@@ -2,4 +2,9 @@ const withBundleAnalyzer = require("@next/bundle-analyzer")({
   enabled: process.env.ANALYZE === "true",
 });
 
-module.exports = withBundleAnalyzer();
+module.exports = withBundleAnalyzer({
```

Comment:
Should we use next-compose plugins? I find the syntax a little easier to understand:

https://github.com/vercel/next.js/tree/canary/packages/next-bundle-analyzer#usage-with-next-compose-plugins

Reply:
i'm torn on this, ideally we're not using so many plugins we need the composer because modules have historically made next.js version bumps difficult for us (tsdc + our scss setup specifically). I agree the syntax of this is odd though.

the plugin compose plugin is not first party, and i can see that plugin itself causing a more difficult upgrade one day: https://github.com/cyrilwanner/next-compose-plugins